### PR TITLE
Disable babel cache compression

### PR DESCRIPTION
**Pull request by ddzz**
## Description
Quick PR that should speed up CI times by ~10-20 seconds.

## Acceptance criteria
- [ ] CI passes.

### CODE REVIEW

**Comment by timwright12** [comment]
Pulling @jhonnyoddball and @alexandec in because we're in the process of upgrading Webpack

File: config/webpack.config.js [hunk 1]
```diff
@@ -293,6 +293,7 @@ module.exports = async (env = {}) => {
             options: {
               // Speed up compilation.
               cacheDirectory: '.babelcache',
+              cacheCompression: false,
```
Comment (jhonnyoddball) [inline config/webpack.config.js:296]: What's the main reason to disabled the `cacheCompression`? would this make the cache bigger but faster?

Comment (ddzz) [reply config/webpack.config.js:296]: Yes, this is just to make it faster.

**Comment by alexandec** [comment]
Can we be sure there won't be any breaking changes and regressions due to upgrading  `css-loader` from 1.0.0 to 3.6.0 and `sass-loader` from 8.0.2 to 10.2.0? Regressions in CSS may not be caught by any of our automated tests

**Comment by ddzz** [comment]
> Can we be sure there won't be any breaking changes and regressions due to upgrading `css-loader` from 1.0.0 to 3.6.0 and `sass-loader` from 8.0.2 to 10.2.0? Regressions in CSS may not be caught by any of our automated tests

Good question. I did a smoke test of a few pages, and I didn't find any obvious CSS issues, but I could've missed something. I upgraded these dependencies to the latest popular versions used with Webpack v4. If we're worried about regressions here I can remove the dependency upgrades from the PR and use this PR only to disable babel cache compression.

**Comment by alexandec** [comment]
Since Jhonny and I will be upgrading all the webpack-related dependencies soon anyway, I think it's safest to do all the upgrades at that point rather than do some of them here.

Regarding disabling cacheCompression, that seems like a good optimization which shouldn't have any negative side effects. I'm good with that change if Jhonny agrees.

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**